src/app/dashboard/reports/page.tsx

```
@@ -0,0 +1,189 @@
+                  </td>
+                  <td className="px-6 py-4 whitespace-nowrap">{report.type}</td>
+                  <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+                    {report.date}
+                  </td>
+                  <td className="px-6 py-4 whitespace-nowrap">
+                    <span className="px-2 inline-flex text-xs leading-5 font-semibold rounded-full bg-green-100 text-green-800">
+                      {report.status}
+                    </span>
+                  </td>
+                  <td className="px-6 py-4 whitespace-nowrap text-sm font-medium">
+                    <a href="#" className="text-indigo-600 hover:text-indigo-900 mr-3">
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)